### DISCUSSION TITLE
onBlur is not getting called if <Input> is under <Tooltip>

### DISCUSSION THREAD

**Discussion by ycaoT** [twilio-labs/paste#1619] [General] [answered]
Looks like a bug. Same thing for TextArea

**Comment by SiTaggart** [comment]
Hi @ycaoT, can you create a Sandbox reproduction for us please. Just so that we know exactly what you're trying to do with both components.

Thanks

**Reply by ycaoT** [reply]
@SiTaggart  Please find the following exmaple:
https://codesandbox.io/s/funny-ritchie-vs4hw?file=/src/App.js

**Reply by nkrantz** [reply]
Hi @ycaoT, just want to let you know that we're working on this!

We've realized the problem is that we're not correctly passing down the props to the Tooltip's child. We'll report back soon with a solution.

**Comment by SiTaggart** [comment, marked as answer]
I've just submitted a Pull Request to address this issue

https://github.com/twilio-labs/paste/pull/1655

One it's merge and the fix released, we'll let you know.

**Reply by gloriliale** [reply]
Update @ycaoT: this fix is live!

**Reply by SiTaggart** [reply]
Almost, it's merged to main. We just need to release it to npm. It's real close though
